http/src/main/scala/org/http4s/blaze/http/package.scala

```diff
@@ -27,7 +35,7 @@ package object http {
       SimpleHttpResponse("OK", 200, headers, ByteBuffer.wrap(body))
 
     def Ok(body: String, headers: Headers): SimpleHttpResponse =
-      Ok(body.getBytes(UTF_8), ("Content-Type", "text/plain; charset=UTF-8")+:headers)
+      Ok(body.getBytes(UTF_8), ("content-type", "text/plain; charset=utf-8")+:headers)
```

Comment:
Why is this lower case?  It's insensitive, but this seems ... non-canonical.  I only ask because I wonder if I missed something important.